Back-To-School/Assets/BackToSchool/Scripts/Interfaces/Core/IGameManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Assets.BackToSchool.Scripts.Parameters;
+using Cysharp.Threading.Tasks;
+
+
+namespace Assets.BackToSchool.Scripts.Interfaces.Core
+{
+    public interface IGameManager
+    {
+        bool    IsSaveDataExists();
+        void    ExitGame();
+        void    StartGame(StartParameters parameters);
+        void    ReturnToMenu();
```

Comment:
Why some of transitions are async and other - not?

Reply:
I mean returning type (Unitask) - you can wait for some of them and cannot for others